db/migrate/20190226182746_create_registration_facets.rb

```diff
@@ -1,5 +1,3 @@
-# frozen_string_literal: true
-
```

Comment:
I'm not sure if this is needed.

Reply:
I don't know why it was added in the first place, but there are no strings in this file so it had no affect anyways.